Makefile

```diff
@@ -60,22 +55,38 @@ DYLIB_MAKE_CMD=$(CC) -shared -Wl,-soname,$(DYLIB_MINOR_NAME)
 STLIBNAME=$(LIBNAME).$(STLIBSUFFIX)
 STLIB_MAKE_CMD=$(AR) rcs
 
+uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+
+# Deps (use make dep to generate this)
+alloc.o: alloc.c fmacros.h alloc.h
```

Comment:
I think this will need to be moved below the `all:` target since right now running make without args only builds `alloc.o`